lib/net/http/responses.rb

```diff
@@ -231,8 +281,11 @@ class HTTPAlreadyReported < HTTPSuccess
   #
   # The <tt>IM Used</tt> response indicates that the server has fulfilled a request
   # for the resource, and the response is a representation of the result
-  # of one or more instance-manipulations applied to the current instance.
```

Comment:
I think you accidentally deleted this line.

Reply:
Fixed.